### PR TITLE
taxonomy: german ingredients

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -76663,6 +76663,7 @@ bg:краставица, краставици
 ca:cogombre
 cs:Okurek
 da:agurk, agurker
+de:Gurke, Gurken
```
Comment (hangy) [inline taxonomies/ingredients.txt:76666]:
```suggestion
de:Gurken, Gurke
```
Categories are usually plural first, with the singular alias following next

**Comment by benbenben2** [comment]
To resolve "Semantic PRs / Validate PR title (pull_request_target)", you have to update the title of your pull request.
Click on Edit
title_edit
And add taxonomy:
image
And save

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (leonore-wilhelm) [reply taxonomies/ingredients.txt:76666]: This is not a category, this is an ingredient :)

Comment (moon-rabbitOFF) [reply taxonomies/ingredients.txt:76666]: This is not a category, this is an ingredient :)

**Review by benbenben2 — APPROVED** [review]
Looks good to me

**Review by hangy — APPROVED** [review]

**Pull request merged**